packages/samples-loctool/webos-c/package.json

```diff
@@ -0,0 +1,27 @@
+{
+    "private": true,
+    "name": "sample-webos-c",
+    "description": "Sample localization project",
+    "repository": "git@github.com:iLib-js/ilib-mono-webos.git",
+    "license": "Apache-2.0",
+    "version": "1.0.0",
+    "scripts": {
+        "loc": "loctool -2 --xliffStyle custom --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB",
+        "debug": "node --inspect-brk node_modules/loctool/loctool.js -2 --xliffStyle custom --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB",
+        "loc-generate": "loctool generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --resourceFileNames json=cstrings.json --plugins webos-c -l en-US,en-AU,en-GB,es-CO,es-ES,fr-CA,fr-FR,ko-KR,ja-JP --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP:en-GB --projectId sample-webos-c",
+        "loc-generate-debug": "node --inspect-brk node_modules/loctool/loctool.js generate -2 -x xliffs --projectType custom --sourceLocale en-KR --resourceFileTypes json=webos-json-resource --plugins webos-c -l en-US,en-AU,en-GB,es-CO,es-ES,fr-CA,fr-FR,ko-KR,ja-JP --localeMap es-CO:es,fr-CA:fr --localeInherit en-AU:en-GB,en-JP:en-GB --projectId sample-webos-c",
+        "clean": "rm -rf *.xliff resources",
+        "sample-test": "node test/testResources.js",
+        "test:all": "npm-run-all clean loc sample-test",
+        "sampleAppTest": "pnpm test:all",
+        "cleanOutput": "rm -rf *.xliff resources"
```

Comment:
Why is cleanOutput needed? 'cleanOutput' is duplicate of 'clean'

Reply:
I wanted to create an additional command that only deletes the generated resource outputs of the sample apps, which is different from the commonly used "clean." So, I added a new command called "cleanOutput"

Reply:
OK